src/components/Footer/Footer.jsx

```diff
@@ -165,21 +165,21 @@ const Footer = ({ strings }) => (
               href="https://play.google.com/store/apps/details?id=com.opendota.mobile&hl=en"
               style={{ position: 'relative', left: '13px', top: '12px' }}
             >
-              <img src="/assets/images/google_play_store.png" alt="" height="46px" />
+              <img src="/assets/images/google_play_store.png" alt="download the android app on google play store" height="46px" />
```

Comment:
Functional links should meet [W3 accessibility standards](https://www.w3.org/WAI/tutorials/images/functional/) by providing alt text that accurately describes what the link does. 

<details><summary>Current behavior</summary>
<img width="1096" alt="Screen Shot 2021-10-02 at 11 15 34 AM" src="https://user-images.githubusercontent.com/15097156/135724568-b109bdac-0301-447d-b849-b079bd570973.png">
</details>

<details><summary>With updated alt text</summary>
<img width="977" alt="Screen Shot 2021-10-02 at 11 20 54 AM" src="https://user-images.githubusercontent.com/15097156/135724611-42aefc33-a291-4474-82e8-53630a7fa625.png">

</details>